uwrt_mars_rover_utils/uwrt_mars_rover_xbox_controller/src/coordinateNode.cpp

```diff
@@ -0,0 +1,28 @@
+#include <uwrt_mars_rover_xbox_controller/coordinateNode.hpp>
+
+namespace drivetraincontrollerComposition
+{
+CoordinateNode::CoordinateNode(const rclcpp::NodeOptions & options)
+: Node("coordinateNode", options)
+{
+  auto callback =
+    [this](const uwrt_mars_rover_xbox_controller::msg::XboxController::SharedPtr msg_in) -> void {
+    //Add scaling value
+    const int vConstant = 5;
```

Comment:
Move this into the hpp file as a constant (so not here). Maybe something like this. It should also be added to the launch file so you can change it from there - but I can do that later so keeping it with the TODO comment is fine:
```suggestion
    // TODO: add a ros param to control the vel const
    const int vel_const_ =  5;
```